yodeploy/test_integration/test_server.py

```diff
@@ -19,9 +18,12 @@ def setUp(self):
         self.username = self.flask_app.config.server.username
         self.password = self.flask_app.config.server.password
 
+        auth_header_val = '%s:%s' % (self.username, self.password)
+        auth_header_val = auth_header_val.encode()
+        auth_header_val = b'Basic %s' % base64.b64encode(auth_header_val)
```

Comment:
Hm, in Python 3.4/3.5, I'm seeing the following. 

```python
'shit'.encode('base64')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
LookupError: 'base64' is not a text encoding; use codecs.encode() to handle arbitrary codecs
```

I also tried it with a byte string, but those don't have encode methods in python 3.